### PR TITLE
Replace solutions cypress test with equivalent pytest

### PR DESCRIPTION
**Pull request by brianpeiris**
### Detailed information:

- Deleted solutions.cy.js and replaced it with an equivalent pytest in app/solutions/tests/test_solutions_routes.py
- Removed obsolete solutionsEndpoint cypress command.
- Moved is_none_or_type helper function to new app/common/tests/test_utils.py file
- Minor improvement to test_myths_routes -- test_myth_properties now asserts that there are some myth objects in the returned array.

### Closing issues: 

List all issues the pull request solve: 

- #405

### Test plan (required)

- [x] The PR contains new PyTest unit/integration tests for any function or functional added. 
- [x] The PR changes existing PyTest unit/integration tests to keep all tests up to date.
- [x] The PR does not lead to degradation in unit test coverage.
- [x] Project parts affected by changes in this PR was tested manually on your local (using Postman or in any other way). List everything you've tested below:
  - Ran pytest. All tests pass, including new tests.
  - Ran cypress. All remaining tests still pass.

### CODE REVIEW

File: cypress/e2e/solutions.cy.js [hunk 1]
```diff
@@ -1,70 +0,0 @@
...
-
-    beforeEach(() => {
-        cy.sessionEndpoint().should((response) => {
-            session_Id = response.body.sessionId
-        }).then(() => {
-            cy.scoresEndpoint(scores, session_Id).should((response) => {
-                set_one_quizId = response.body.quizId;
-            });
-        });
-    });
```
Comment (brianpeiris) [inline cypress/e2e/solutions.cy.js:18]: The cypress test had this beforeEach setup, but as far as I can tell, it didn't actually have any bearing on these tests in the first place (removing this from the cypress test still allowed the test to pass), so I did not replicate it in the pytest.

File: app/common/tests/test_utils.py [hunk 2]
```diff
@@ -0,0 +1,2 @@
+def is_none_or_type(value, expected_type):
```
Comment (danmash) [inline app/common/tests/test_utils.py:1]: DRY 👍

**Review by danmash — APPROVED** [review]
Sorry for the delay, 
Awesome work on this 👍 Keep going 🚀

File: cypress/e2e/solutions.cy.js (hunk 1 above)
Comment (danmash) [reply cypress/e2e/solutions.cy.js:18]: np

**Pull request merged**